whole-tale/girder_wholetale
# Add an Endpoint for Importing a DataONE Dataset

This PR adds an endpoint for taking a redirect from DataONE and directs the user to the proper location on the dashboard (currently, the compose page). I also renamed a few files in lib.dataone.


To Test:
   1. Deploy this branch
   2. Visit `https://girder.local.wholetale.org/api/v1#!/integration/integration_dataoneDataImport`
   3. Set `uri` to `urn:uuid:test.12345`
   4. Set `title` to `Dataset Title`
   5. Set `environment` to `Jupyter Notebook`
   6. Click `Try it out!`
   7. Navigate to the URL that was produced
   8. Note that the behavior should be expected

I matched the `apiToken` parameter that is present in the Dataverse implementation. It may come in handy if the user wants to try the feature with private data.

I'll squash the commits on merge.
